docs/development/how-to/ide-setup/intellij-setup.md

```diff
@@ -32,3 +45,4 @@ Note: The below instructions reference IntelliJ Settings, which are accessible v
   1. *File > Import Settings*
```

Comment:
On my Linux system (version ideaIC-2021.2) importing settings is found under `File > Manage IDE Settings > Import Setting`